tests/browser/xhr/ajax-events.browser.js

```diff
@@ -224,6 +224,59 @@ test('prepareHarvest correctly serializes an AjaxRequest events payload', functi
   t.end()
 })
 
+test('prepareHarvest correctly serializes an AjaxRequest events that is lacking an endTime', function (t) {
+  const context = { spaNode: undefined }
+  const expected = {
```

Comment:
This expected object is not used for any assertions - should it be? Or, is the test intended to only test that the event is serialized without checking values?